docs/assets/diagrams/linking/1-create-listener.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
+title Create Listener
+
+autonumber
+participant App
+database Firebase
+participant "PISP Server" as PISP
+
+PISP -> Firebase: ""firebase.firestore()""\n\
+  "".collection("consents")""\n\
+  "".where("status", "!=", "SUCCESS")""\n\
```

Comment:
```suggestion
  "".where("status", "!=", "ACTIVE")""\n\
```

Reply:
if this is meant to be : 
*select all revoked consents* ?

Reply:
or even all non-ACTIVE consents

I realize you're using `consent.status` for a lot more than its intended to be used for 
in general - it only has 2 possible properties - `ACTIVE` and `REVOKED` - one tells you its valid and ready to be used - the other that its been revoked and cannot be used anymore

for all the other uses we might want to rely more on checking credential status? or creating a sep field associated with consent in PISP demo server

Reply:
Actually, we use `status` field as a state indicator for the linking process. If we are going to have a status for `ACTIVE` and `REVOKED`, we probably need to have another field called `state` which I think is a more representative name to avoid confusion. But, yeah, we need the current enumeration of `status` to determine the which phase we are currently in for the linking.

Reply:
resolving this issue based on our discussion in the call